homeassistant/components/soundtouch/media_player.py

```diff
@@ -23,6 +25,9 @@
 SERVICE_ADD_ZONE_SLAVE = 'soundtouch_add_zone_slave'
 SERVICE_REMOVE_ZONE_SLAVE = 'soundtouch_remove_zone_slave'
 
+SERVICE_SNAPSHOT = 'soundtouch_snapshot'
```

Comment:
We should no longer add services under the media player domain. Instead, register them under `soundtouch`.